packages/lesswrong/components/tagging/subforums/SubforumSubforumTab.tsx

```diff
@@ -240,7 +240,7 @@ const SubforumSubforumTab = ({tag, userTagRel, isSubscribed, classes}: {
             fragmentName: "PostsRecentDiscussion",
             render: (post: PostsRecentDiscussion) => {
               // Remove the intro post from the feed IFF it has not been dismissed from the top
-              return post._id !== tag.subforumIntroPost?._id && !hideIntroPost && (
+              return !(post._id === tag.subforumIntroPost?._id && hideIntroPost) && (
```

Comment:
This was meant to filter out the intro post from the main feed, unless it had been dismissed. But it actually filtered out all posts in the case where the intro post had been dismissed